scripts/pi-hole/js/utils.js

```diff
@@ -561,35 +561,35 @@ function listAlert(type, items, data) {
   // Show a list of successful items if there are any
   if (data.processed.success.length > 0) {
     message +=
-      "<strong>Successfully added " +
```

Comment:
This and further HTML tags are removed as this string is later HTML-encoded ans they have been appearing in the toast as plain text. This is already broken in `development-v6` right now.